common/src/main/java/com/twilio/video/examples/common/VideoFrameKtx.kt

```diff
@@ -0,0 +1,171 @@
+package com.twilio.video.examples.common
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.ImageDecoder
+import android.graphics.ImageFormat
+import android.graphics.Matrix
+import android.graphics.Rect
+import android.graphics.YuvImage
+import android.os.Build
+import tvi.webrtc.VideoFrame
+import tvi.webrtc.YuvConverter
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.nio.ByteBuffer
+
+
+/**
+ * Converts a [tvi.webrtc.VideoFrame] to a Bitmap. This method must be called from a thread with a
+ * valid EGL context when the frame buffer is a [VideoFrame.TextureBuffer].
+ */
+fun VideoFrame.toBitmap() : Bitmap? {
+    return captureBitmap(this)
+}
+
+private fun captureBitmap(frame: VideoFrame) : Bitmap? {
```

Comment:
Nit pick but you can use a [single-expression function](https://kotlinlang.org/docs/reference/functions.html#single-expression-functions) here.